src/SDKs/CognitiveServices/dataPlane/Vision/Face/Face/Customizations/EmotionExtensions.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.CognitiveServices.Vision.Face.Models;
+
+namespace Microsoft.Azure.CognitiveServices.Vision.Face
+{
+    /// <summary>
+    /// Extension methods for Emotion.
+    /// </summary>
+    public static partial class EmotionExtensions
+    {
+        /// <summary>
+        /// Create a sorted key-value pair of emotions and the corresponding scores, sorted from highest score on down.
+        /// Sorting criteria: Score is the primary key, and the name is the secondary key.
```

Comment:
I would update the comment to say the secondary key is sorted alphabetically.

Reply:
I agree, I have already updated the comment.